### ISSUE TITLE
babel-plugin-normalize-requires removes `.js` extension from folder names under certain circumstances

### ISSUE THREAD

**Issue by izaera** [liferay/liferay-npm-build-tools#14]
See discussion at https://github.com/liferay/liferay-npm-build-tools/pull/11, and specially the comment https://github.com/liferay/liferay-npm-build-tools/pull/11#issuecomment-314427293.

**Comment by jbalsas** [comment]
I guess this should be apparent if trying to use something like [highlight.js](https://highlightjs.org/) or [chart.js](http://www.chartjs.org/)?

**Comment by yuchi** [comment]
Yeah, Highlight.js was indeed the one I checked for to verify it actually had `.js` in the name.

**Comment by izaera** [comment]
See another possible strategy in https://github.com/liferay/liferay-npm-build-tools/pull/12#issuecomment-314676518

**Comment by izaera** [comment]
Please note that packages with trailing `.js` in their names (whether or not they are scoped) should work already because of https://github.com/liferay/liferay-npm-build-tools/pull/12.

Nevertheless, we will provide an example with highlight.js and/or chart.js to check it.

**Comment by wincent** [comment]
Going to do a mass-transfer of issues (including this one):

- **From:** https://github.com/liferay/liferay-js-toolkit/issues
- **To:** https://github.com/liferay/liferay-frontend-projects/issues

Note that many of the issues that we move may be stale, but we can close those once we've had time to review them. In the meantime, doing a bulk move makes is cheap and it will allow us to mark the liferay-js-toolkit repo as archived (read-only), which is an important final step in the consolidation process in order to avoid confusion.
